src/testdrive/src/action/s3.rs

```diff
@@ -24,6 +27,7 @@ use rusoto_sqs::{
     ReceiveMessageRequest, SetQueueAttributesRequest, Sqs,
 };
 
+use crate::action::file::{build_compression, Compression as Comp};
```

Comment:
It's a personal taste thing, and we do it elsewhere in the codebase so it's up to you, but I find renaming items like this confusing.

My style would be to get rid of both the imports, and disambiguate by just writing out `flate2::Compression` and `file::Compression` as necessary.

Reply:
While I would probably do what Brennan recommended, the other common thing to do is to make the rename longer rather than shorter: `use flate2::Compression as Flate2Compression`.

Reply:
Thinking about this overnight, I think I can remove the `file::Compression` part entirely by merging the two match statements below (the first is superfluous).

Reply:
Please ignore my last comment -- I was thinking of the S3 download code. I'll make the rename longer, thanks!